test/FactorialServiceSpec.js

```diff
@@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
     
     it('should null if an invalid param was given', function() {
         expect(FactorialService.calculate(-1)).toBe(null);
-        expect(FactorialService.calculate("1")).toBe(null);
```

Comment:
Please take a look about the rules to calculate the factorial! It only
apply for positive and natural number.

https://pt.m.wikipedia.org/wiki/Fatorial

On Wed, Jun 15, 2016, 12:51 AM Guilherme Sanches Pereira <
notifications@github.com> wrote:

> In test/FactorialServiceSpec.js
> <https://github.com/univas/js-factorial/pull/3#discussion_r67096069>:
>
> > @@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
> >
> >      it('should null if an invalid param was given', function() {
> >          expect(FactorialService.calculate(-1)).toBe(null);
> > -        expect(FactorialService.calculate("1")).toBe(null);
>
> Yeah, but this caused a problem with number float. How is done parse to
> int, if to write 4.5, the code sends only 4, generating result 24... which
> theoretically should not occur. Anyway. Thanks for the answers.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/univas/js-factorial/pull/3/files/6ed811756193b59df50f2c558299a651336358c5#r67096069>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFE-fh9gOR0_PCSfXCDJIyjT_xefCh5qks5qL3a9gaJpZM4I1l8i>
> .
>